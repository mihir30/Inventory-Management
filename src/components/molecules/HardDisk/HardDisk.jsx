import React from 'react';
const HardDisk = ({setComputerModel}) => {

    return (
            <div className='group2'>
                <label className='form-label'>
                    Size,Gb
                    <span className='star-mark'>*</span>
                </label>
                <input className='form-control' type='number' defaultValue={160}/>
            </div>
    )
}
export default HardDisk;