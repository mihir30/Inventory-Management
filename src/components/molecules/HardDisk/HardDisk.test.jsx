import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDom from "react-dom";
import HardDisk from './HardDisk';

describe("Hard disk Component", () => {
    it('renders the component properly', () => {
        const divEl = document.createElement("div");
        ReactDom.render(<BrowserRouter><HardDisk setComputerModel={"PC-00019"}></HardDisk></BrowserRouter>, divEl); 
        
    });


    it('should have the right hard disk component snapshot', () => {
        const tree = renderer.create().toJSON();
        expect(tree).toMatchSnapshot();
    })

});