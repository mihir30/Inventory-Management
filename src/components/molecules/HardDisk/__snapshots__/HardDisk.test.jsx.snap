// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hard disk Component should have the right hard disk component snapshot 1`] = `null`;
